src/extension_helpers.ts

```diff
@@ -0,0 +1,22 @@
+import * as vscode from "vscode";
+
+export function get_build_directory(): string {
+  let build_directory: string = "";
+  let config = vscode.workspace.getConfiguration("cmake");
+  if (config.has("buildDirectory")) {
+    build_directory = config.get("buildDirectory") as string;
+  } else {
+    config = vscode.workspace.getConfiguration("ctest-lab");
+    build_directory = config.get("buildDirectory") as string;
+  }
+  return replace_code_variables(build_directory);
+}
+
+function replace_code_variables(path: string): string {
+  if (vscode.workspace.workspaceFolders) {
+    const workspaceFolder: string =
+      vscode.workspace.workspaceFolders[0].uri.fsPath;
+    return path.replace("${workspaceFolder}", workspaceFolder);
+  }
+  return path;
+}
```

Comment:
Did you copy these functions without modifying them?

Reply:
Yes, just moved them so they are accessible to both _test\_*.ts_ files